docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    restart: always
+    networks:
+      - potatonotes
+    environment:
+      POSTGRES_USER: potatonotes
+      POSTGRES_DB: potatonotes
+      POSTGRES_PASSWORD: potatonotes
+    volumes:
+    - db-data:/var/lib/postgresql/data
+
+  api:
+    image: atechnohazard/potatosync:latest
+    restart: on-failure
+    networks:
+      - potatonotes
+    depends_on:
+      - db
+    environment:
```

Comment:
I would prefer if you used the .env file for populating the container env.

Reply:
Hmm i can do that however it would require people to create a .env file before they can run docker-compose up. This way they only need to download 1 file, run 1 command and they are set.